NetKAN/RealPlume-StockConfigs.netkan

```diff
@@ -0,0 +1,32 @@
+{
+    "spec_version"      : "v1.4",
+    "$kref"             : "#/ckan/github/KSP-RO/RealPlume-StockConfigs",
+    "ksp_version"       : "1.0",
+    "name"              : "Real Plume - Stock Configs",
+    "identifier"        : "RealPlume-StockConfigs",
+    "abstract"          : "A set of effects and basic configs that can be attached to any engine as appropriate.  Requires secondary configs to function correctly.",
+    "license"           : "CC-BY-NC-SA",
+    "depends" : [
+        { "name" : "ModuleManager" },
+        { "name" : "SmokeScreen" },
+        { "name" : "RealPlume" }
+    ],
+    "suggests" : [
+        { "name" : "ABLaunchers" },
+        { "name" : "FASA" },
+        { "name" : "KWRocketry" },
+        { "name" : "SpaceY-Lifters" },
+        { "name" : "TantaresLV" },
+        { "name" : "VenStockRevamp" }
+    ],
+    "provides"  : [ "RealPlumeConfigs" ],
+    "resources" : {
+        "homepage" : "https://github.com/KSP-RO/RealPlume/wiki"
+    },
+    "install" : [
+        {
+            "find"       : "RealPlume-Stock",
+            "install_to" : "GameData"
```

Comment:
Currently you'll have GameData/RealPlume and GameData/RealPlume-Stock. Not a big deal but it might be confusing to users. If possible, consider `"install_to" :GameData/RealPlume"` -- Ultimately your choice though.